### DISCUSSION TITLE
Attempting to use "Frame" mode slideshow on the Unsorted album results in Base64 complaint error

### DISCUSSION THREAD

**Discussion by jessienab** [LycheeOrg/Lychee#2648] [Bugs] [answered]
### Lychee version

v6.0.1

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

Attempting to use the "Frame" slideshow mode on the default "Unsorted" Smart Album, results in Lychee failing to launch the slideshow in that mode, and complains about the lack of a Base64 encoded ID for the album:

Example: https://photos . nabein . me/gallery/unsorted

Attempting to select the Computer icon in upper right (Frame slideshow mode) results in this error:

> album id must be either null or a string in Base64-encoding with 24 characters

Full error:
```
Illuminate\Foundation\Http\FormRequest::failedValidation:168 album id must be either null or a string in Base64-encoding with 24 characters
App\Http\Requests\BaseApiRequest::validateResolved:80 album id must be either null or a string in Base64-encoding with 24 characters; caused by
```

HTTP error:
```
GET
	
scheme
	https
host
	photos.nabein.me
filename
	/api/v2/Frame
album_id
	unsorted
Address

Status
422
Unprocessable Entity
VersionHTTP/2
Transferred3.45 kB (197 B size)
Referrer Policyno-referrer

	
cache-control
	private, must-revalidate

content-type
	application/json
cross-origin-embedder-policy
	unsafe-none
cross-origin-opener-policy
	unsafe-none
cross-origin-resource-policy
	cross-origin
date
	Tue, 05 Nov 2024 02:12:12 GMT
expires
	-1
permissions-policy
	accelerometer=(self), ambient-light-sensor=(self), attribution-reporting=*, autoplay=(self), bluetooth=(self), browsing-topics=*, camera=(self), compute-pressure=(self), cross-origin-isolated=(self), display-capture=(self), document-domain=*, encrypted-media=(self), fullscreen=(self), gamepad=(self), geolocation=(self), gyroscope=(self), hid=(self), identity-credentials-get=(self), idle-detection=(self), local-fonts=(self), magnetometer=(self), microphone=(self), midi=(self), otp-credentials=(self), payment=(self), picture-in-picture=*, publickey-credentials-create=(self), publickey-credentials-get=(self), screen-wake-lock=(self), serial=(self), speaker-selection=(self), storage-access=*, usb=(self), web-share=(self), window-management=(self), xr-spatial-tracking=(self)
pragma
	no-cache
referrer-policy
	no-referrer
server
	nginx
set-cookie
strict-transport-security
	max-age=63072000
x-content-type-options
	nosniff
x-download-options
	noopen
X-Firefox-Spdy
	h2
x-frame-options
	deny
x-permitted-cross-domain-policies
	none
x-xss-protection
	1; mode=block
	
Accept
	application/json, text/plain, */*
Accept-Encoding
	gzip, deflate, br
Accept-Language
	en-US,en;q=0.5
cache-control
	no-cache
Connection
	keep-alive
Content-Type
	application/json
Cookie

expires
	0
Host
	photos.nabein.me
pragma
	no-cache
Sec-Fetch-Dest
	empty
Sec-Fetch-Mode
	cors
Sec-Fetch-Site
	same-origin
TE
	trailers
User-Agent
	Mozilla/5.0 (X11; Linux x86_64; rv:115.0) Gecko/20100101 Firefox/115.16.1
X-Requested-With
	XMLHttpRequest
X-XSRF-TOKEN


### Steps to reproduce the issue

1. Go to https://photos . nabein . me/gallery/unsorted
2. Select the Computer icon upper right
3. Nothing should display. You'll see a blank screen
4. Press ESC to return to the album, you'll see the Base64 error as a red banner at the top.

### Diagnostics [REQUIRED]

```
Errors
warning
lossless_optimization set to 1 but svgo not found!
warning
zend.assertions is disabled although Lychee is in debug mode. For easier debugging code generation for assertions should be enabled.
info
Latest version of PHP is 8.3
warning
Online updates are disabled by configuration
info
Found 8 medium that could be generated.
info
Have you considered supporting Lychee? :)
Info

Lychee Version (tag):                      v6.0.1 (6fc3e35) -- Up to date (-5.0862506341958E-11 years ago).
DB Version:                                6.0.1
composer install:                          --no-dev
APP_ENV:                                   production
APP_DEBUG:                                 true
APP_URL:                                   set
APP_DIR:                                   default
LOG_VIEWER_ENABLED:                        true
VUEJS_ENABLED:                             true
PHOTO_PIPES:                               true
System:                                    Linux
PHP Version:                               8.2.24
PHP User agent:                            Lychee/4 (https://lycheeorg.github.io/)
Timezone:                                  America/Toronto
Max uploaded file size:                    100M
Max post size:                             100M
Chunk size:                                25.60 MB
Max execution time:                        3600
MySQL Version:                             10.11.9-MariaDB-ubu2204
exec() Available:                          yes
Imagick Available:                         1
Imagick Enabled:                           1
Imagick Version:                           1691
GD Version:                                2.3.3
Number of foreign key:                     12 found.

		

Space

Lychee total space:                        208.52 MB
		

Configuration

version:                                   060001
check_for_updates:                         0
sorting_photos_col:                        taken_at
sorting_photos_order:                      ASC
sorting_albums_col:                        title
sorting_albums_order:                      ASC
imagick:                                   1
skip_duplicates:                           0
small_max_width:                           0
small_max_height:                          360
medium_max_width:                          1920
medium_max_height:                         1080
lang:                                      en
image_overlay_type:                        desc
default_license:                           CC0
compression_quality:                       90
grants_full_photo_access:                  1
delete_imported:                           0
mod_frame_enabled:                         1
mod_frame_refresh:                         30
thumb_2x:                                  1
small_2x:                                  1
medium_2x:                                 1
landing_page_enable:                       0
force_migration_in_production:             0
search_public:                             1
hide_version_number:                       1
SL_enable:                                 0
SL_for_admin:                              0
recent_age:                                7
grants_download:                           1
photos_wraparound:                         1
map_display:                               1
zip64:                                     1
map_display_public:                        1
map_provider:                              OpenStreetMap.org
force_32bit_ids:                           0
map_include_subalbums:                     1
update_check_every_days:                   3
has_exiftool:                              1
share_button_visible:                      0
import_via_symlink:                        0
has_ffmpeg:                                1
location_decoding:                         0
location_decoding_timeout:                 30
location_show:                             1
location_show_public:                      1
rss_enable:                                0
rss_recent_days:                           7
rss_max_items:                             100
prefer_available_xmp_metadata:             0
editor_enabled:                            1
lossless_optimization:                     1
swipe_tolerance_x:                         150
swipe_tolerance_y:                         250
local_takestamp_video_formats:             .avi|.mov
log_max_num_line:                          1000
unlock_password_photos_with_url_param:     0
nsfw_visible:                              1
nsfw_blur:                                 1
nsfw_warning:                              1
nsfw_warning_admin:                        1
nsfw_banner_override:                      
map_display_direction:                     1
album_subtitle_type:                       oldstyle
upload_processing_limit:                   6
new_photos_notification:                   0
legacy_id_redirection:                     1
zip_deflate_level:                         9
default_album_protection:                  1
album_decoration:                          layers
album_decoration_orientation:              row
allow_username_change:                     0
auto_fix_orientation:                      1
use_job_queues:                            0
random_album_id:                           starred
use_last_modified_date_when_no_exif_date:  0
layout:                                    justified
date_format_photo_thumb:                   M j, Y, g:i:s A e
date_format_photo_overlay:                 M j, Y, g:i:s A e
date_format_sidebar_uploaded:              M j, Y, g:i:s A e
date_format_sidebar_taken_at:              M j, Y, g:i:s A e
date_format_hero_min_max:                  F Y
date_format_hero_created_at:               M j, Y, g:i:s A T
date_format_album_thumb:                   M Y
upload_chunk_size:                         0
nsfw_banner_blur_backdrop:                 0
search_pagination_limit:                   25
search_minimum_length_required:            3
photo_layout_justified_row_height:         320
photo_layout_masonry_column_width:         300
photo_layout_grid_column_width:            250
photo_layout_square_column_width:          200
photo_layout_gap:                          12
display_thumb_album_overlay:               always
display_thumb_photo_overlay:               hover
default_album_thumb_aspect_ratio:          1/1
use_album_compact_header:                  0
login_button_position:                     left
back_button_enabled:                       0
back_button_text:                          Return to Home
current_job_processing_visible:            1
SA_random_thumbs:                          0
job_processing_queue_visible:              1
login_required:                            0
enable_unsorted:                           1
enable_starred:                            1
enable_recent:                             1
enable_on_this_day:                        1
login_required_root_only:                  1
dark_mode_enabled:                         1
show_keybinding_help_popup:                0
show_keybinding_help_button:               1
disable_se_call_for_actions:               0
enable_se_preview:                         0
slideshow_timeout:                         5
keep_original_untouched:                   1
hide_nsfw_in_smart_albums_and_search:      1

		
```

### Browser & System [REQUIRED]

IceCat 115.16.1esr (64-bit) (Firefox ESR)
Firefox 131.0.2 (64-bit)
Librewolf 131.0.2-1

Linux 6.6.56-1-lts #1 SMP PREEMPT_DYNAMIC Thu, 10 Oct 2024 12:04:53 +0000 x86_64 GNU/Linux

Arch Linux

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
Fix incomming: https://github.com/LycheeOrg/Lychee/pull/2683
